setup.py

```diff
@@ -159,7 +159,8 @@ def cmd_config(self, about, allow_empty=False):
         author_email = 'lgautier@gmail.com',
         requires = requires,
         install_requires = requires + ['cffi>=1.0.0'],
-        setup_requires = ['cffi>=1.0.0'],
+        tests_require= requires + ['pytest', 'numpy', 'pytz', 'pandas', 'ipython', 'tzlocal'],
+        setup_requires = ['cffi>=1.0.0', 'pytest-runner'],
```

Comment:
I am on the fence about this because I am not familiar with `pytest-runner`. Unless a compelling use-case (or I can be convinced that running `python setup.py test` instead of `pytest [blah]` is a compelling use-case), I'd be happy to merge the PR without it.

Reply:
the compelling use case would be that `python setup.py test` would automatically download and install the setup and test requirements in an isolated environment.

Reply:
At this point I'd rather let a user who want this install `pytest-runner` on his own rather than add it as a dependency for the whole project then.